ui/desktop/app/styles/app.scss

```diff
@@ -530,3 +530,111 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     background-color: var(--token-color-palette-neutral-50);
   }
 }
+
+.details-screen-body {
+  .rose-layout-body-content {
+    margin-top: 1.5rem;
+  }
+}
+
+// Session Details - Credentials Panel
+.credential-panel-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 0.5rem;
+
+  .title {
+    font-weight: var(--token-typography-font-weight-bold);
+  }
+
+  /* stylelint-disable selector-class-pattern */
+  .hds-form-field--layout-flag {
+    margin-bottom: 0;
+  }
+}
+
+.credential-panel-body {
+  border: 0.063rem solid var(--token-color-border-primary);
+  border-radius: 0.313rem;
+  display: flex;
+  margin-top: 0.75rem;
+
+  .credential-header {
+    padding: 1.5rem;
+    width: 33%;
+  }
+}
+
+.credential-secret {
+  width: 66%;
+  margin: 0.75rem;
+
+  .secret-container {
+    margin: 0.75rem 0.5rem;
+    border-bottom: 0.0625rem solid var(--token-color-border-primary);
+
+    &:last-child {
+      border-bottom: 0;
+    }
+
+    .secret-key {
+      font-weight: var(--token-typography-font-weight-semibold);
+    }
+
+    .hidden-secret {
+      margin: 0.25rem 0;
+
+      .rose-button-ghost {
+        margin-bottom: 0;
+        color: var(--ui-gray-subtler-2);
+      }
+    }
+  }
+}
+
+.credential-name {
+  font-weight: var(--token-typography-font-weight-semibold);
+}
+
+.raw-credential-panel-body {
+  $border-width: 0.063rem;
+
+  border: 0.063rem solid var(--token-color-border-primary);
+  border-radius: 0.313rem;
+  display: block;
+  margin-top: 0.75rem;
+
+  .raw-credential-header {
+    padding: 1.5rem 1.5rem 0.75rem;
+  }
+
+  .raw-credential-title {
+    display: flex;
+    justify-content: space-between;
+
+    .hds-copy-button {
+      padding: 0;
+      background: unset;
+      border: unset;
+      box-shadow: unset;
+      min-height: unset;
```

Comment:
Should we be overriding these styles or just adhering to the standard HDS styling for this?

Reply:
I originally left it with the standard HDS styling. I decided to change it so that the copy buttons matched with the `<HiddenSecret/>` component. I was going to remove this if we have time to update all of the copy buttons to the HDS version. I can address this in another PR asap if you'd like but I didn't want the scope to grow too much for this PR.

Reply:
It is okay if the HDS copy buttons aren't consistent with Boundary's old copy buttons. Ideally, we should use the hds style and try not to override the styles. We can create a ticket in the backlog to update the rest of the copy buttons when we get time, this is the approach we followed when migrating components.

Reply:
Overrides removed.